Releases/LatestRelease/integration-AlienVaultOTX.md

```diff
@@ -0,0 +1 @@
+- bug fix hash file type in human readable.
```

Comment:
@GalRabinDemisto RN are in old format, please follow new convention:
https://github.com/demisto/content/tree/master/docs/release_notes

Reply:
Hi,
@bakatzir 
Just change the release notes by the new convention.

Thanks for the review.